lib/IO/Socket/SSL.rakumod

```diff
@@ -22,89 +22,153 @@ has Int $.ins = 0;
 
 has $.client-socket;
 has $.listen-socket;
-has $.accepted-socket;
-has $!socket;
-has OpenSSL $.ssl;
```

Comment:
It looks like these changes will require a major version bump. It might be worth mentioning in the issue which methods have gone away to make it easier to update the github release notes later.

Reply:
The only bits of the public API I removed are `$.accepted-socket` and `$.ssl`. I'll mention it in the PR.

Reply:
@ugexe Talking about github releases. This module currently still lives in P6C. When introducing an incompatible version bump and expecting users to bump their dependencies (or continue using the old version), then P6C is a bad fit. Do you think it would be feasible to move the module to fez?

Reply:
Ideally it would be on fez, yes. I kind of wonder if this should be moved to the raku community modules now, since otherwise it is kind of weird for the github repo to be owned by an entity who is not associated with the releases. I'm not sure taking ownership of this under my account is ideal (i.e. an auth of `zef:ugexe`) either, so if not raku community modules then we'd need to create some other fez group to release under 🤔